libs/filter_example/src/graph_utils.rs

```diff
@@ -51,12 +51,16 @@ pub fn generate_target_graph(
  *
```

Comment:
This double maintenance of graph_utils seems pointless. We might as well make this a lib in the simulator. Then the filter and this example filter can import it and only one file has to be curated.

Reply:
Makes sense.  I refactored it into its own library in the simulator and deleted it from the compiler.  Now it'll only be in one place.